Week01/Problem03/nkovacs_03.py

```diff
@@ -0,0 +1,13 @@
+def factors(n):
+	output = []
+	f = 2
+	while f * f <= n:
+		if n % f == 0:
+			output.append(f)
+			n = n / f
+		else:
+			f = f + 1
+	output.append(int(n))
+	return(output)
+
+print(factors(600851475143)[-1])
```

Comment:
What's the absolute largest value that any factor of a number can be? How could you use that knowledge to reduce the amount of work you're doing in the loop inside the `factors` function?